sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverClient.java

```diff
@@ -36,10 +41,10 @@
     private static final ReceiveAsyncOptions DEFAULT_RECEIVE_OPTIONS = new ReceiveAsyncOptions()
         .setEnableAutoComplete(false)
         .setMaxAutoRenewDuration(Duration.ZERO);
+    private static ContinuesMessageSubscriber continuesMessageSubscriber = null;
```

Comment:
Why static? If someone else creates another instance of receiver, they will get this. Which will be subscribed to another instance.